app/scripts/metamask-controller.js

```diff
@@ -4062,6 +4055,17 @@ export default class MetamaskController extends EventEmitter {
     this.sendUpdate();
   }
 
+  //=============================================================================
+  // REQUEST QUEUE
+  //=============================================================================
+
+  setUseRequestQueue = (value) => {
```

Comment:
this looks like it could be a static method, we should use this syntax the rest of the controller uses:

```suggestion
  setUseRequestQueue(value) {
```